x/storage/keeper/keeper.go

```diff
@@ -1508,6 +1525,7 @@ func (k Keeper) getDiscontinueObjectStatus(ctx sdk.Context, objectId types.Uint)
 	store := ctx.KVStore(k.storeKey)
 	bz := store.Get(types.GetDiscontinueObjectStatusKey(objectId))
 	if bz == nil {
+		ctx.Logger().Error("fail to get discontinued object status", "object id", objectId)
```

Comment:
Having delete logic(#1532 `store.Delete(types.GetDiscontinueObjectStatusKey(objectId))` ) inside the GetXXX function is not considered a good practice.

Reply:
renamed the function to make it clear.